static/scss/partials/landing/a.scss

```diff
@@ -61,3 +77,363 @@
+        }
+
+    }
+
+    .c-pricing-text {
+        grid-row-start: 1;
+        grid-column-start: span 3;
+        height: 100%;
+        text-align: center;
+
+        h2 {
+            font-family: $font-metropolis;
+            font-weight: 500;
+            font-size: 2.5em;
+        }
+
+        .c-text {
+            padding: 0 $spacing-lg;
+        }
+
+        @media screen and ($mq-lg) {
+            text-align: start;
+            grid-column-start: span 2;
+            grid-row-start: auto;
+
+            .c-text {
+                padding: 0;
+                display: flex;
+                flex-direction: column;
+                justify-content: center;
+                height: 100%;
+            }
+        }
+    }
+}
+
+.use-case-section {
+    box-shadow: 0px 15px 15px $color-light-gray-20;
+}
+
+.c-use-case-content {
+    display: grid;
+    grid-template-columns: 1fr;
+    row-gap: $spacing-md;
+   
+    .use-case-title {
+        padding: $spacing-md $spacing-md $spacing-sm $spacing-md;
+        display: flex;
+        font-family: $font-metropolis;
+        border-radius: $border-radius-md;
+    }
+
+    .use-case-title-text {
+        color: $color-black;
+        font-weight: bold;
+        border-radius: $border-radius-md;
+        font-family: $font-metropolis;
+        font-size: 1.125em;
+        text-align: start;
+        align-self: flex-end;        
+    }
+
+    .use-case-shopping {
+        background-color: $color-yellow-20;
+    }
+
+    .use-case-social-networks {
+        background-color: $color-yellow-40;
+    }
+
+    .use-case-offline {
+        background-color: $color-green-40;
+    }
+
+    .use-case-access-content {
+        background-color: $color-red-20;
+    }
+
+    .use-case-gaming {
+        background-color: $color-purple-30;
+    }
+
+    .use-case-desc {
+        display: none;
+        background: $color-light-gray-10;
+        color: $color-black;
+        border-radius: $border-radius-md;
+        padding: $spacing-lg;
+    }
+
+    .desc-headline {
+        font-weight: bold;
+        font-size: 1.125em;
+    }
+    
+    .is-active + .use-case-desc {
+        display: inline;
+    }
+
+    //Desktop view
+    @media screen and ($mq-lg) {
+        grid-template-columns: repeat(5, 1fr);
+        column-gap: $spacing-md;
+
+        .use-case-title {
+            float: left;
+            height: 180px;
+            width: 180px;
+            border-radius: $border-radius-md;
+            grid-row-start: 1;
+            padding-bottom: $spacing-xs;
+        }
+
+        .use-case-title-text {
+            text-align: center;
+            padding-bottom: $spacing-xs;
+            width: 100%;    
+        }
+
+        .is-active + .use-case-desc {
+            display: flex;
+            flex-direction: row;
+            grid-row-start: 2;
+            margin-top: $spacing-md;
+            grid-column: span 5;
+        }
+
+        .desc-headline {
+            font-size: 1.25em;
```

Comment:
Use the [typography sizing](https://protocol.mozilla.org/fundamentals/typography.html#code) from Protocol to set the size.